blazor/pdfviewer/how-to/load-office-files.md

```diff
@@ -0,0 +1,144 @@
+---
+layout: post
+title: Load office files in Blazor PDF Viewer | Syncfusion
+description: Learn here all about how to load the microsoft office files like powerpoint in the Syncfusion Blazor PDF Viewer component and more.
```

Comment:
Description: Learn how to load the Microsoft office files like PowerPoint in the Syncfusion Blazor PDF Viewer component and more.